pdns/protobuf.cc

```diff
@@ -97,6 +97,40 @@ void DNSProtoBufMessage::setEDNSSubnet(const Netmask& subnet, uint8_t mask)
 #endif /* HAVE_PROTOBUF */
 }
 
+void DNSProtoBufMessage::addTag(const std::string& strValue)
+{
+#ifdef HAVE_PROTOBUF
+
+  PBDNSMessage_DNSResponse* response = d_message.mutable_response();
+  if (!response)
+    return;
+
+  response->add_tags(strValue);
+
+#endif /* HAVE_PROTOBUF */
+}
+
+void DNSProtoBufMessage::addRR(const std::string& strName, uint32_t uType, uint32_t uClass, uint32_t uTTL, const uint8_t *ptrBlob, size_t uBlobLen)
```

Comment:
`uType` and `uClass` should be `uint16_t`, `ptrBlob` and `uBlobLen` can be merged into a `std::string`.